src/background/services/network/utils/getSyncDomain.ts

```diff
@@ -5,6 +5,9 @@ const SYNCED_DOMAINS = [
   'core.app',
   'test.core.app',
   runtime.id,
+  'develop.avacloud-app.pages.dev',
+  'avacloud.io',
+  'staging--ava-cloud.avacloud-app.pages.dev',
```

Comment:
I think we should have a separate list of the getAddressesInRange list, since we don't want to give the same ability for all RPCs for AvaCloud. Like account renaming, deletion or creation should not be exposed to them. Also this way if the network is set to BTC in Core, Avacloud would need to switch to BTC as well, which caused issues in the past.